Assets/Scripts/EnemyScripts/Shockwave.cs

```diff
@@ -0,0 +1,66 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Shockwave : MonoBehaviour
+{
+    public GameObject player;
+    private int shockwaveDamage;
+    public bool playerHit;
+      public int ShockwaveDamage
+    {
+        get { return shockwaveDamage; }
+        set { shockwaveDamage = value; }
+    }
+
+    private int shockwaveSize;
+    public int ShockwaveSize
+    {
+        set
+        {
+            shockwaveSize = value;
+            transform.localScale = new Vector3(shockwaveSize, shockwaveSize, shockwaveSize);
+        }
+    }
+
+    void Start()
+    {
+
+        shockwaveDamage = 30 + (GameSettings.waveNumber / 5) * 5;
```

Comment:
This is the same as 30 + GameSettings.waveNumber
IF this is intentional it can be simplified, otherwise it needs to be fixed.